cli/internal/cmd/upgradeplan.go

```diff
@@ -92,7 +104,9 @@ func upgradePlan(cmd *cobra.Command, planner upgradePlanner, patchLister patchLi
 	if err != nil {
 		return fmt.Errorf("calculating next image minor version: %w", err)
 	}
-
+	up.log.Debugf(`Current image minor version is %s
+current CLI minor version is %s
+next image minor version is %s`, currentImageMinorVer, currentCLIMinorVer, nextImageMinorVer)
```

Comment:
```suggestion
	up.log.Debugf("Current image minor version is %s", currentImageMinorVer)
	up.log.Debugf("Current CLI minor version is %s", currentCLIMinorVer)
	up.log.Debugf("Next image minor version is %s", nextImageMinorVer)
```
For log formatting reason, prefer to write a new message instead of introducing new lines